SwiftProtobuf.xcodeproj/project.pbxproj

```diff
@@ -2136,6 +2142,9 @@
 				SDKROOT = appletvos;
 				TARGETED_DEVICE_FAMILY = 3;
 				TVOS_DEPLOYMENT_TARGET = 9.0;
+				VALID_ARCHS = "";
+				"VALID_ARCHS[sdk=appletvos*]" = arm64;
+				"VALID_ARCHS[sdk=appletvsimulator*]" = "x86_64 i386";
```

Comment:
Why 32bit for simulators but only 64bit for device?

Reply:
That's a great question. Do you know if the simulators will always be 64bit?